addons/marketing_card/models/card_template.py

```diff
@@ -0,0 +1,16 @@
+from odoo import fields, models
+
+# Good ratio to have a large image still small enough to stay under 5MB (common limit)
+# Close to the 2:1 ratio recommended by twitter and these dimensions are recommended by meta
+# https://developers.facebook.com/docs/sharing/webmasters/images/
+# https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary-card-with-large-image
+TEMPLATE_DIMENSIONS = (1200, 630)
+TEMPLATE_RATIO = 40 / 21
+
+class CardCampaignTemplate(models.Model):
+    _name = 'card.template'
+    _description = 'Marketing Card Template'
+
+    name = fields.Char(required=True, translate=True)
+    default_background = fields.Image()
+    body = fields.Html(sanitize=False)
```

Comment:
Sooo, about that 😄 

Should we make this model strictly manageable by the super admin? To lessen security concerns?
Or do we expect people to create those?

Maybe it's less of a concern as this is only ran through to make an image out of, not entirely sure. 
Security team will probably have a more educated opinion on it.